assets/app/scripts/controllers/builds.js

```diff
@@ -44,7 +44,43 @@ angular.module('openshiftConsole')
     watches.push(DataService.watch("buildConfigs", $scope, function(buildConfigs) {
       $scope.buildConfigs = buildConfigs.by("metadata.name");
       Logger.log("buildConfigs (subscribe)", $scope.buildConfigs);
-    }));    
+    }));
+
+    var associateRunningBuildToBuildConfig = function(buildsByBuildConfig) {
+      var buildConfigBuildsInProgress = {}; 
+      for (var buildConfigName in buildsByBuildConfig) {
+        var buildConfigBuilds = buildsByBuildConfig[buildConfigName];
```

Comment:
its a style thing that we eliminate doing this in two lines by using angular.forEach instead